<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-3K67UQ0M" ignored="0" name="GTPL Banking Edge Test" tests="1" failures="1" timestamp="26 Feb 2019 05:09:09 GMT" time="12.559" errors="0">
  <testcase name="loginTest" time="0.576" classname="com.gtplbank.testCases.TC_UserLoginTest_001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.gtplbank.testCases.TC_UserLoginTest_001.loginTest(TC_UserLoginTest_001.java:34)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
</testsuite> <!-- GTPL Banking Edge Test -->
